python_modules/dagster/dagster/_core/storage/upath_io_manager.py

```diff
@@ -274,7 +293,7 @@ def _load_partition_from_path(
             context (InputContext): IOManager Input context
             partition_key (str): the partition key corresponding to the partition being loaded
             path (UPath): The path to the partition.
-            backcompat_path (Optional[UPath]): The path to the partition in the backcompat location.
+            backcompat_path (Optional[UPath]): The path to the partition in the backcompat location. These paths can be present in assets produced by very old Dagster versions (TODO: specify concrete version).
```

Comment:
do you intend to resolve these TODOs before landing?

Reply:
Oh yeah, I'll find the exact version

Reply:
i just merged it as is - can merge a PR that adds this if you find the exact version